src/macros.jl

```diff
@@ -505,9 +495,13 @@ Add a group of semidefinite constraints described by the expression `expr`
 parametrized by `i`, `j`, ...
 
 The expression `expr` needs to be of the form `a sign b` where `sign` is `⪰`,
-`≥`, `>=`, `⪯`, `≤` or `<=` and `a` and `b` are `square` matrices. It
-constrains the matrix `x = a - b` (or `x = b - a` if the sign is `⪯`, `≤` or
-`<=`) to be symmetric and positive semidefinite.
+`≥`, `>=`, `⪯`, `≤` or `<=` and `a` and `b` are `square` matrices. It constrains
+the square matrix `x` (or `-x` if the sign is `⪯`, `≤` or `<=`) to be symmetric
+and positive semidefinite where
+
+* if `b` is the symbol `0`, `x = a`,
```

Comment:
"where if" is hard to read. Consider reversing as: "`x = a`, if `b` is the symbol `0`,"